app/models/conversation_account.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+# == Schema Information
+#
+# Table name: conversation_accounts
+#
+#  id                      :bigint(8)        not null, primary key
+#  account_id              :bigint(8)
+#  conversation_id         :bigint(8)
+#  participant_account_ids :bigint(8)        default([]), not null, is an Array
+#  status_ids              :bigint(8)        default([]), not null, is an Array
+#  last_status_id          :bigint(8)
+#
+
+class ConversationAccount < ApplicationRecord
```

Comment:
I think AccountConversations makes at least a *little* bit more sense here.

Reply:
You're right, that makes perfect sense, and I hate how much I'm gonna need to search and replace here oof